lib/src/sql/number.rs

```diff
@@ -522,7 +522,7 @@ pub trait Sort {
 
 impl Sort for Vec<Number> {
 	fn sorted(&mut self) -> Sorted<&Vec<Number>> {
-		self.sort();
+		self.sort_unstable();
 		Sorted(self)
 	}
 }
```

Comment:
This is a slight semantic change for performance reasons. If the input is `[Number::Int(5), Number::Float(5.0)]`, the sorted array may now be `[Number::Float(5.0), Number::Int(5)]`.